### ISSUE TITLE
Cannot find module use-inertia-form or its corresponding type declarations

### ISSUE THREAD

**Issue by tasso85** [aviemet/useInertiaForm#27]
I have just installed this by using the following command:

```
npm install use-inertia-form --save-dev
```

From what I can see, this installed latest 4.3.1 version.

Then, I simply tried replace one use of `useForm` with `useInertiaForm` after `import`ing it; however, both the IDE and Vite now show this error:

```
TS2307: Cannot find module use-inertia-form or its corresponding type declarations.
```

Did I perhaps simply install/import it wrong? Or is this problem caused by something else?

**Comment by tasso85** [comment]
If it is relevant, I would like to add that, if I open the folder `node_modules/use-inertia-form` I see only 3 files:
- LICENSE
- package.json
- README.md

**Comment by tasso85** [comment]
I solved this by rolling back to 4.3.0; after doing so, in the folder now there is also a `dist` subfolder, and the import works properly, maybe there is something wrong specifically with version 4.3.1

**Comment by aviemet** [comment]
Hello, sorry for the delay in seeing this. Pretty confusing behavior, unfortunately I'm unable to reproduce it. I've made some changes to the rollup config though to address some bundle size issues, hopefully that also addresses this issue.
